### DISCUSSION TITLE
Converted Github account to organisation - Lost Supabase account access

### DISCUSSION THREAD

**Discussion by sudayavarman** [supabase/supabase#21101] [Troubleshooting]
_This is a copy of a troubleshooting article on Supabase's docs site. It may be missing some details from the original. View the [original article](https://supabase.com/docs/guides/troubleshooting/converted-github-account-to-organisation---lost-supabase-account-access-5wsE_1)._

[Converting](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization) your GitHub account to a [GitHub organization](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts#organization-accounts) will cause you to lose access to all other accounts using GitHub sign-ins.

If possible, create a new GitHub account using the **same email** as the previous GitHub account that was turned into a Organization or that was deleted. Once you sign in with that newly created account, you should have access to the old account. You can then invite any new account to have access to the Organization and Projects as needed.

If that is not possible, Supabase can help you transfer ownership of an existing organization on your account by inviting a new owner to your organization. This new user will need to have a different email from the previous one, and you will need to email [support@supabase.com](email:support@supabase.com) with that new email as well to confirm that you own the email address.

**Comment by Simon-1985** [comment]
Hi I have accidentally converted my Github account to an organisation which appears to have lose access to all other accounts. Can this be reversed? I have emailed support@supabase.com but been advised to also message here.

**Comment by lucas-ault** [comment]
I've run into this issue and am a paid supabase user - haven't received a response on my ticket and need to get back into my account asap. Any other methods to contact support?

**Reply by GaryAustin1** [reply]
Email is only support contact. This info I got from SB more recently on this particular issue 



Turns out the fix for these "I converted to a Github Org" situations is:
For your issue, you can re-create a Github account using the same email as the Github account that was turned into a Organization. 
Once you sign in with that newly created account, you should have access to the old account. 
You can then invite the new account to have access to the Organization and Projects as needed.

**Comment by devpjnovak007** [comment]
The worst technical support - where is the anwer?
